assets/text/effects/wildermyth-drauven-pcs_outpostCelebration.properties

```diff
@@ -0,0 +1,6 @@
+#suppress inspection "UnusedProperty" for whole file
+.longName=Outpost Celebration
+.name=
+~01~~panel_001~1_narration=One night, the Drauven and humans came together to enjoy a night of merry making and commraderie.
+~01~~panel_001~2_hero2=Dang, <hero>, this party is something else! What exactly is it for anyway?
+~01~~panel_001~3_hero=To show our efforts have born fruit, my friend!
```

Comment:
This might be a good place to add at a personality split for the two heroes - what do you think?